config/schema.error.yaml

```
@@ -0,0 +1,575 @@
+    title: 'Configuration is not compatible with this version of Magento'
+    stage: build
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_SCD_OPTIONS_IGNORANCE:
+    title: 'Scd options ignored'
+    stage: build
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_CONFIGURATION_STATE_NOT_IDEAL:
+    title: 'The configured state is not ideal'
+    stage: build
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_BALER_CAN_NOT_BE_USED:
+    title: 'Baler JS bundling cannot be used'
+    stage: build
+    step: run-baler
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_REDIS_SERVICE_NOT_AVAILABLE:
+    title: 'Cache is configured for a Redis service that is not available. Configuration will be ignored.'
+    stage: deploy
+    step: 'pre-deploy:cache'
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_WRONG_SPLIT_DB_CONFIG:
+    title: 'The configured state is not ideal'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_DIR_NESTING_LEVEL_NOT_CONFIGURED:
+    title: 'The directory nesting level value for error reporting has not been configured'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_NOT_CORRECT_LOCAL_XML_FILE:
+    title: 'Invalid file config ./pub/errors/local.xml'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_ADMIN_DATA_IGNORED:
+    title: 'Admin data is used to create an admin user during initial installation only and is ignored during upgrade process'
+    suggestion: 'You can remove admin data from configuration'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_ADMIN_EMAIL_NOT_SET:
+    title: 'Admin user was not created as admin email was not set'
+    suggestion: 'Create an admin user via ssh manually: bin/magento admin:user:create'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_UPDATE_PHP_VERSION:
+    title: 'Update php version to recommended'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_SOLR_DEPRECATED:
+    title: 'Solr support has been deprecated in Magento 2.1.'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_SOLR_NOT_SUPPORTED:
+    title: 'Solr is no longer supported by Magento 2.2 or later.'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_ES_INSTALLED_BUT_NOT_USED:
+    title: 'Elasticsearch service is installed at infrastructure layer but is not used as a search engine.'
```

```suggestion
    title: 'Elasticsearch service is installed at infrastructure layer, but it is not used as a search engine.'
```

"not used as a search engine" -- because it's not configured?